helm/templates/cron-watcher.yaml

```diff
@@ -0,0 +1,66 @@
+---
```

Comment:
Hm, I believe this stuff is already placed in the iac repo.
See https://github.com/glifio/filecoin-iac/blob/main/k8s/cronjob_cid-checker.tf

Reply:
Well, I mixed up 2 things - watcher and checker, sorry.

Reply:
But we can move it into iac as future improvement